.github/workflows/edge.yml

```diff
@@ -7,14 +7,8 @@ name: edge
 
 on:
   workflow_dispatch:
-  push:
-    branches:
-      - main
-    paths-ignore:
-      - '**/**.md'
-      - '**/**.asciidoc'
-      - 'docs/**'
-      - 'examples/**'
+  schedule:
+    - cron: '0 6 * * 1,4'
```

Comment:
Note for reviewers: Monday and Thursday 6AM. Both cases there's time to inspect if there is a build failure but I have no real preferences and want to know your thoughts

Reply:
Twice per week sounds great.

nit: Could perhaps have a comment saying this is twice per week, for those of us that don't speak cron very well. :)